twitter.js

```diff
@@ -49,9 +61,69 @@ class Twitter {
     };
 
     this.url = getUrl(config.subdomain);
+    this.oauth = getUrl(config.subdomain, "oauth");
     this.config = config;
   }
+  
+  async getRequestToken(twitterCallbackUrl) {
+    // return `oauth_token` & `oauth_token_secret`
```

Comment:
Think we should remove these comments they're helpful but this stuff can be covered in the README.

Reply:
And the same comments apply for the getAccessToken method.